src/app/components/charts/chart.service.js

```diff
@@ -335,12 +396,17 @@ angular.module('transmartBaseUi')
                     .y(height * set.legH)
                     .itemHeight(4 + 6 * set.min/300)
```

Comment:
10 px is the base height for the item
300 is the starting size for a chart
set.min is the smallest of height or width

so: 
4 + 6 * 300/300 = 10 px

It could also be  10 * set.min/300 but this way it becomes too small to quick so i put it in the form:

A + Bx 
where:
A = min px size = 4
B = to make result 10 when chart is 300px
x = size/baseSize